server-reboot/server.sh

```diff
@@ -1,209 +1,54 @@
-#!/bin/sh
```

Comment:
This line, at least, should stay.  Also, some kind of block comment telling what the file does...